frontend/src/components/RegionModal.jsx

```diff
@@ -84,11 +84,12 @@ export default function RegionModal({
 		const updatedRegion = { name: regionName, description };
 		const requestUrl = `/regions/${selectedRegionID || ""}`;
 		const requestMethod = selectedRegionID ? http.put : http.post;
+		console.log("SELECTED REGION ID" + selectedRegionID);
 
 		requestMethod(requestUrl, updatedRegion)
 			.then((response) => {
 				log.info("Region saved successfully: ", response.data);
-				updateRegionsList(response.data);
+				updateRegionsList(response.data.new_ent | response.data.updated);//TODO: This is just bad. The whole method should be 2 methods. Clever but confusing.
```

Comment:
Do you know what you could do then...?

Reply:
... make it 2 methods?

Reply:
YES! HOLY MOLY BENNETT

Reply:
This is a refactor, which we are trying not to do at this point. Diverting to #2 . We'll just use the same pattern we did for editing sites... or will we... I think we're considering making a RegionManagment page. which would be nice.